src/components/reusables/ActionBar/ActionBar.tsx

```diff
@@ -77,7 +77,7 @@ export const ActionBar = ({
             </button>
           ) : (
             <button
-              className={cn(classes.actionBarHeader)}
+              className={cn(classes.actionBarHeader, classes.clickable)}
```

Comment:
```suggestion
              className={cn(classes.actionBarHeader, { [classes.clickable]: onClick })}
```

For the cases where it isn't in fact clickable

Reply:
Oh yeah right hehe, on to another task so a bit hard to context switch